src/app/league/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class LeagueNotFound(Exception):
```

Comment:
i prefer calling these ``` LeagueNotFoundException ``` so that it's immediately clear they are exception objects